ports/jasper/vcpkg.json

```
@@ -19,5 +14,26 @@
       "name": "vcpkg-cmake-config",
       "host": true
     }
-  ]
+  ],
+  "default-features": [
+    "freeglut",
+    "opengl"
+  ],
+  "features": {
+    "freeglut": {
+      "description": "Open source implementation of GLUT with source and binary backwards compatibility.",
+      "dependencies": [
+        {
+          "name": "freeglut",
+          "platform": "!osx"
+        }
+      ]
+    },
+    "opengl": {
+      "description": "OpenGL (Open Graphics Library) is the computer industry's standard API for defining 2-D and 3-D graphic images.",
+      "dependencies": [
+        "opengl"
+      ]
+    }
+  }
```

This is inaccurate; `opengl` and `freeglut` should be the same feature:

```suggestion
  "default-features": [
    "opengl"
  ],
  "features": {
    "opengl": {
      "description": "Enable the use of the OpenGL/GLUT Library",
      "dependencies": [
        {
          "name": "freeglut",
          "platform": "osx"
        },
        "opengl"
      ]
    }
  }
```